### PR TITLE
FIX: import *.txn file with txhex - extra newline characted prevented…

### LINKED ISSUE #2161: Import of the file failed with a transaction-signed.
There is import of the file that quote (signed transaction) is broken the wallet cannot do
the decode is to broadcast to the network, I need to copy to the hex-transaction and send
by Network/Broadcast.

### PR DESCRIPTION
**Pull request by Overtorment**
… it from being recognized (closes #2161)

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/2q4zc5k5n5qpm59rwedjm7bq4r

[download apk](http://filestorage.bluewallet.io:1488/1606495814.apk)

**Review by marcosrdz — APPROVED** [review]

File: screen/send/details.js [hunk 1]
```diff
@@ -922,7 +922,8 @@ export default class SendDetails extends Component {
         this.setState({ isLoading: false, isAdvancedTransactionOptionsVisible: false });
       } else if (DeeplinkSchemaMatch.isTXNFile(res.uri)) {
         // plain text file with txhex ready to broadcast
-        const file = await RNFS.readFile(res.uri, 'ascii');
+        const file = (await RNFS.readFile(res.uri, 'ascii')).replace('\n', '').replace('\r', '');
+        console.warn(JSON.stringify(file));
```
Comment (limpbrains) [inline screen/send/details.js:926]: Do you want to keep this?

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

File: screen/send/details.js (hunk 1 above)
Comment (Overtorment) [reply screen/send/details.js:926]: oops I thought I deleted it

Comment (marcosrdz) [reply screen/send/details.js:926]: I saw it and was like "...maybe its intentional since ive done it"

**Comment by Overtorment** [comment]
See it, say it, sorted

**Pull request merged**